### ISSUE TITLE
Improved description how to enable the filter in uBO

### ISSUE THREAD

**Issue by davidhedlund** [lassekongo83/Frellwits-filter-lists#12]
https://github.com/lassekongo83/Frellwits-filter-lists/blob/master/README.md reads:
> Copy the raw link above for the filter list you want to subscribe to. How to import external filter lists in uBlock Origin.

No need to do that since your fortunately is part of uBlock Origin:

![screenshot from 2018-03-20 15 15 38](https://user-images.githubusercontent.com/6416073/37659956-944802f0-2c51-11e8-90e9-705ddc9ab16a.png)

So you just need to describe how to enable it with something like:
> Open the dashboard, go to **3rd party-filters**, and scroll down to **Regions, languages** to enable the **SWE: Frellwit's Swedish Filter​​​​​​​​​**.

**Comment by davidhedlund** [comment]
Thank you.

**Comment by davidhedlund** [comment]
I improved the description slightly. Can you please add this:

> This Sweden's regional filter in [uBlock Origin](https://github.com/gorhill/uBlock) (uBO) located in: **Dashboard** -> **3rd party-filters** -> **Regions, language** -> **SWE: Frellwit's Swedish Filter**
>
> uBO enables the region/language-filter that corresponds to the locale of your web browser. This means that **SWE: Frellwit's Swedish Filter** will be enabled by default if your browser has Swedish locale installed, and that you have to enable the filter manually as describe above otherwise.

**Comment by stanlane** [comment]
Superb.

**Comment by davidhedlund** [comment]
Great.

Don't you think the **How to subscribe** section should be removed in https://github.com/lassekongo83/Frellwits-filter-lists/blob/master/README.md ?

**Comment by davidhedlund** [comment]
Sorry, let me rephrase: don't you think the text should be move to the **How to subscribe** section in https://github.com/lassekongo83/Frellwits-filter-lists/blob/master/README.md ?

**Comment by lassekongo83** [comment]
Tog bort den delen och satte in den texten där den behövs istället.

**Comment by davidhedlund** [comment]
Det ser bra ut. Tack åter igen.

**Comment by davidhedlund** [comment]
aftonbladet.se är helt reklamfritt nu. Tack så jättemycket för det!

**Comment by davidhedlund** [comment]
Jag har väntat över ett år på att någon skulle ta över underhåll av det Svenska filtret så ditt bidrag kommer som ett paket på posten.
